Calorie Tracker/CalorieController.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  CalorieController.swift
+//  Calorie Tracker
+//
+//  Created by Alex Thompson on 2/22/20.
+//  Copyright © 2020 Lambda School. All rights reserved.
+//
+
+import Foundation
+import CoreData
+import UIKit
+
+class CalorieController {
+    func create(calorie: Int16) {
```

Comment:
`calorie` refers to an Int here, rather than the object, which can be confusing. Consider renaming to `create(with calorieAmount` to be more clear?